pom.xml

```diff
@@ -14,10 +14,9 @@
   <description>An Institutional Repository IIIF manifest generation service.</description>
 
   <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.5.4.RELEASE</version>
-    <relativePath /> <!-- lookup parent from repository -->
+    <groupId>edu.tamu.weaver</groupId>
+    <artifactId>webservice-parent</artifactId>
+    <version>2.0.0</version>
```

Comment:
Should reference the weaver messaging module snapshot. The CI will pass when that snapshot is deployed to our maven repository.

Reply:
https://github.com/TAMULib/IRIIIFService/commit/597c61eeeced7c98b4fe7fb925840de2c64df1b5